src/phantom.js

```diff
@@ -107,6 +107,15 @@ export default class Phantom {
      * Returns a value in the global space of phantom process
      * @returns {Promise}
      */
```

Comment:
I'm sorry, indeed, I changed the readme file, but I completely forgot the
doc this time...

Yes, that would be a good idea I think! So people could use their own
external loggers for more customisation.

2016-08-20 0:31 GMT+04:00 Amir Raminfar <notifications@github.com>:

> In src/phantom.js
> <https://github.com/amir20/phantomjs-node/pull/529#discussion_r75547056>:
>
> > @@ -107,6 +107,15 @@ export default class Phantom {
> >       * Returns a value in the global space of phantom process
> >       * @returns {Promise}
> >       */
>
> Maybe a better option would be to do
>
> this.logger = logger
>
> in the constructor so that others can do whatever they want.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amir20/phantomjs-node/pull/529/files/fbe85d868c49dba9a6ccf18bde18699105e46fb8#r75547056>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAu9falkwAiQfkMDIxW0zmZ8zMhtipJ9ks5qhhK1gaJpZM4Joy6d>
> .
>